linkerd/docs/namer.md

```diff
@@ -821,7 +821,7 @@ would be rewritten to `/pfx/host/port/etc`.
 /$/io.buoyant.porthostPfx/<prefix>/<host>:<port>/etc
 ```
 
-Rewrites a name of the form "host:ip" as a path with ip followed by host. Does
+Rewrites a name of the form "host:port" as a path with ip followed by the port. Does
```

Comment:
I think we want "as a path with port followed by host"

Reply:
Yes, definitely. I'll update that tomorrow.